zenodo/modules/records/templates/zenodo_records/record_detail.html

```diff
@@ -155,4 +155,8 @@ <h5>{{_('Thesis supervisor(s)')}}</h5>
 </script>
 <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-5137aff66ad9c2a1"></script>
 <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
+{% set jsonld_serialization = record | serialize_record(pid, 'schemaorg_jsonld_v1')%}
+<script type='application/ld+json'>
+{{ jsonld_serialization|safe }}
```

Comment:
@lniensen is it OK to pipe the JSON (string) from a serializer output through ``safe`` here?

Reply:
I think it should be safe, since it will be a valid JSON encoding - i.e. no escaping necessary.